ppl/zqd/handler.go

```diff
@@ -29,6 +30,7 @@ func NewHandler(core *Core, logger *zap.Logger) http.Handler {
 	h.Use(accessLogMiddleware(logger))
 	h.Use(panicCatchMiddleware(logger))
 	h.Handle("/ast", handleASTPost).Methods("POST")
+	h.Router.Handle("/metrics", promhttp.HandlerFor(core.registry, promhttp.HandlerOpts{}))
```

Comment:
Because h.Handle would invoke handler.Handle (https://github.com/brimsec/zq/pull/1627/files/8397b9cf18f9845b3d109bc5ea6f907e3ab14096#diff-d33b860101636265e3225d3197e58fb916e20e11afd430aadf1f748a59fbfb9aL20-R25).

Reply:
What do you think about declaring the mux above, then adding the prom route, and perhaps the `/status`, `/version`, and `/` routes on the mux directly?

Reply:
Feels out of scope here, but I do that at https://github.com/brimsec/zq/tree/zqd-personalities.

Reply:
👍